docs/zh_cn/tutorials/customize_datasets.md

```diff
@@ -171,3 +171,38 @@ data = dict(
 )
 
 ```
+
+### 多图混合集
+
+我们使用 `MultiImageMixDataset` 作为包装(wrapper)去混合多个数据集的图片。
+`MultiImageMixDataset`可以被类似mosaic和mixup的多图混合数据増广使用。
+
+`MultiImageMixDataset`与`Mosaic`数据増广一起使用的例子：
+```python
```

Comment:
MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines